src/probnum/randprocs/markov/integrator/_iwp.py

```diff
@@ -0,0 +1,318 @@
+        Optional. Default is :math:`d=1`.
+        The dimension of the integrated Wiener process itself is :math:`d(\nu + 1)`.
+    initrv
+        Law of the integrated Wiener process at the initial time point.
+        Optional. Default is a :math:`d(\nu + 1)` dimensional standard-normal distribution.
+    diffuse
+        Whether to instantiate a diffuse prior. A diffuse prior has large initial variances.
+        Optional. Default is `False`.
+        If `True`, and if an initial random variable is not passed, an initial random variable is created,
+        where the initial covariance is of the form :math:`\kappa I_{d(\nu + 1)}`
+        with :math:`\kappa=10^6`.
+        Diffuse priors are used when initial distributions are not known.
+        They are common for filtering-based probabilistic ODE solvers.
+    forward_implementation
+        Implementation of the forward-propagation in the underlying transitions.
+        Optional. Default is `classic`. `sqrt` implementation is more computationally expensive, but also more stable.
+    backward_implementation
+        Implementation of the backward-conditioning in the underlying transitions.
+        Optional. Default is `classic`. `sqrt` implementation is more computationally expensive, but also more stable.
+
+    Raises
+    ------
+    Warning
+        If `initrv` is not None and `diffuse` is True.
+
+    Examples
+    --------
+    >>> iwp1 = IntegratedWienerProcess(initarg=0.)
+    >>> print(iwp1)
+    <IntegratedWienerProcess with input_dim=1, output_dim=2, dtype=float64>
+
+    >>> iwp2 = IntegratedWienerProcess(initarg=0., num_derivatives=2)
+    >>> print(iwp2)
+    <IntegratedWienerProcess with input_dim=1, output_dim=3, dtype=float64>
+
+    >>> iwp3 = IntegratedWienerProcess(initarg=0., wiener_process_dimension=10)
+    >>> print(iwp3)
+    <IntegratedWienerProcess with input_dim=1, output_dim=20, dtype=float64>
+
+    >>> iwp4 = IntegratedWienerProcess(initarg=0., num_derivatives=4, wiener_process_dimension=1)
+    >>> print(iwp4)
+    <IntegratedWienerProcess with input_dim=1, output_dim=5, dtype=float64>
+    """
+
+    def __init__(
+        self,
+        initarg,
+        num_derivatives=1,
+        wiener_process_dimension=1,
+        initrv=None,
+        diffuse=False,
+        forward_implementation="classic",
+        backward_implementation="classic",
+    ):
+        iwp_transition = IntegratedWienerTransition(
+            num_derivatives=num_derivatives,
+            wiener_process_dimension=wiener_process_dimension,
+            forward_implementation=forward_implementation,
+            backward_implementation=backward_implementation,
+        )
+        if initrv is not None and diffuse:
+            warnings.warn(
+                "Parameter `diffuse` has no effect, because an `initrv` has been provided."
+            )
+        if initrv is None:
+            if diffuse:
+                scale_cholesky = 1e3
```

Comment:
As somebody who is not proficient in this whole diffusion domain: Did you potentially mean to write `1e-3` here?